### PR TITLE
Decorate the Cookies section

### PR DESCRIPTION
**Pull request by annevk**
This is roughly where things stand today after having discussed 1-3 of https://github.com/privacycg/meetings/issues/16.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,13 +30,22 @@ For some user agent state (Cookies and Storage below in particular are under dis
...
 
-A likely inexhaustive enumeration of user agent state and ongoing standards activity:
+This section contains a likely inexhaustive enumeration of user agent state and ongoing standards activity. If there is state or standards activity missing please [file an issue](https://github.com/privacycg/storage-partitioning/issues/new) or provide a pull request.
+
+### Cookies
+
+The tentative overall plan is to block cross-site cookies and add support for partitioned cookies via opt-in. The details are still under discussion though it seems likely to be eventually standardized across the IETF and WHATWG. Relevant discussions:
+
+* [Cookie layering](https://github.com/httpwg/http-extensions/issues/2084) (a start of a discussion with the IETF how to best structure the HTTP State Management Mechanism specification to account for these changes to cookies)
+* For opt-in partitioned cookies [CHIPS](https://github.com/WICG/CHIPS) is the most likely candidate, though a few favor [an approach using `requestStorageAccess()`](https://github.com/privacycg/storage-access/issues/75).
```
Comment (johannhof) [inline README.md]: How about:

```suggestion
* For opt-in partitioned cookies [CHIPS](https://github.com/WICG/CHIPS) is the most likely candidate, though a few favor [an approach using `requestStorageAccess()`](https://github.com/privacycg/storage-access/issues/75). These two approaches are not necessarily conflicting and some browsers have expressed interest in supporting both.
```

File: README.md [hunk 2]
```diff
@@ -30,13 +30,22 @@ For some user agent state (Cookies and Storage below in particular are under dis
...
+* Meeting proposal: [cross-site cookies standardization, part 2](https://github.com/privacycg/meetings/issues/19).
```
Comment (johannhof) [inline README.md]:
```suggestion
* Meetings: 
  * [Cross-site cookies standardization](https://github.com/privacycg/meetings/issues/16) ([minutes](https://github.com/privacycg/meetings/blob/main/2022/telcons/04-28-minutes.md)).
  * [cross-site cookies standardization, part 2](https://github.com/privacycg/meetings/issues/19) ([minutes](https://github.com/privacycg/meetings/blob/main/2022/telcons/05-12-minutes.md)).
```

(I hope the Markdown is correct, I couldn't preview it)

**Review by johannhof — APPROVED** [review]
Thanks!

File: README.md [hunk 3]
```diff
@@ -16,11 +16,11 @@ User agent state that is keyed by a single [origin](https://html.spec.whatwg.org
 
 To solve a key aspect of this, any such user agent state needs to be keyed by more than a single origin or site.
```
Comment (Black1230) [inline README.md:17]: activar

**Review by Black1230 — APPROVED** [review]

**Pull request merged**